### PR TITLE
Lint: max-len 100

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
Did ~150 corrections of lines surpassing 100, breaking them mostly below 80.

### CODE REVIEW

File: ts/Core/Series/SeriesType.d.ts [hunk 1]
```diff
@@ -41,7 +41,10 @@ export type SeriesTypeOptions = SeriesType['options'];
  * All possible series plotOptions.
  */
 export type SeriesTypePlotOptions = {
-    [K in keyof SeriesTypeRegistry]?: Omit<SeriesTypeRegistry[K]['prototype']['options'], NonPlotOptions>;
+    [K in keyof SeriesTypeRegistry]?: Omit<
+    SeriesTypeRegistry[K]['prototype']['options'],
+    NonPlotOptions
```
Comment (KacperMadej) [inline ts/Core/Series/SeriesType.d.ts:46]: Why not 1 more indent for the lines from between `<` and `>` ?

Comment (TorsteinHonsi) [reply ts/Core/Series/SeriesType.d.ts:46]: I agree, but the linter wouldn't allow it... Looks like a bug to me.

Comment (TorsteinHonsi) [reply ts/Core/Series/SeriesType.d.ts:46]: BTW, sorry for tagging you in the weekend. Didn't mean to, I must have accidentally clicked your name :)

Comment (KacperMadej) [reply ts/Core/Series/SeriesType.d.ts:46]: (😃 Don't worry, I wasn't tagged. It's still a weekday for me (but not for long) - just going through issues and this poped-up.)

**Pull request merged**